### PR TITLE
Refactore leaderboards and add them in langing page also

### PR DESCRIPTION
**Pull request by alexmcode**
I\m not sure if the positioning in the landing page is optimal

### CODE REVIEW

**Comment by smcgregor** [comment]
* These leaderboard blocks need some explanation for what information they are presenting. Maybe just a header for `Incident Report Submission Leaderboards`.

* Can you make the current leaderboard into search links as well?

* Let's also create a new `about` page that contains all the textual content from the current landing page, then let's make the landing page have exclusively the content you are developing. I will add a little bit of introductory content as well that links to the new `about` page.

**Comment by alexmcode** [comment]
>Let's also create a new about page that contains all the textual content from the current landing page, then let's make the landing page have exclusively the content you are developing. I will add a little bit of introductory content as well that links to the new about page.

@smcgregor I think we should leave this point after we merge all #135 tasks. it will make the commit history much more readable and it will be also easier for us to develop. WHat do you think? does it make sense?

File: site/gatsby-site/pages/summaries/leaderboard.js [hunk 1]
```diff
@@ -1,89 +1,15 @@
...
-        {content.map((value) => (
-          <li key={`${title}-${value[0]}`}>
-            {value[0]}: {value[1]}
-          </li>
-        ))}
-      </ul>
-    </>
-  );
-};
+import SubmittersLeaderboards from 'components/leaderboards/SubmittersLeaderboards';
```
Comment (smcgregor) [inline site/gatsby-site/pages/summaries/leaderboard.js:7]: `SubmittersLeaderboards` -> `SubmittersLeaderboard`

File: site/gatsby-site/src/components/leaderboards/Leaderboard.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+  for (const item in dataHash) {
+    sortedArray.push({
+      label: dataHash[item].fieldValue,
+      value: dataHash[item].totalCount,
+      attribute,
+    });
+  }
+
+  sortedArray = sortedArray.sort((a, b) => {
```
Comment (smcgregor) [inline site/gatsby-site/src/components/leaderboards/Leaderboard.js:15]: Does graphql have an order-by property that would allow for avoiding this sorting?

**Comment by smcgregor** [comment]
This looks great. I left 2 inline comments. One for consistency in naming and another potential tweak. Otherwise this looks good.

File: site/gatsby-site/src/components/leaderboards/Leaderboard.js (hunk 2 above)
Comment (alexmcode) [reply site/gatsby-site/src/components/leaderboards/Leaderboard.js:15]: No, it doesn't seem to have an order bu for group aggregations. I think the performance is good enough as it is. If we really want to have them already sorted in the client we could add a new mongo collection

**Comment by smcgregor** [comment]
Implementation is excellent. Can you put the blocks into a 3 column responsive layout? A basic card layout for visual separation would also be helpful. Once that is good I will compose this MR with the other one for the search box and the complete updating of the landing page and merge.

**Comment by smcgregor** [comment]
Closing in favor of: https://github.com/PartnershipOnAI/aiid/pull/148

**Pull request closed without merging**